crates/extensions/c8y_config_manager/src/config.rs

```diff
@@ -95,6 +95,23 @@ impl ConfigManagerConfig {
             mqtt_port,
             tedge_http_address,
             tedge_http_port,
-        ))
+        );
+        is_config_dir_exist(config.plugin_config_path.as_path())?;
+        Ok(config)
+    }
+}
+
+fn is_config_dir_exist(config_dir: &Path) -> Result<(), TEdgeConfigError> {
+    match config_dir.parent() {
+        Some(cdir) => {
```

Comment:
The name of the parameter is confusing, as this is the path to the config file and not the config directory.

Reply:
Agree, renamed it.